geniza/corpus/templates/corpus/snippets/pagination.html

```diff
@@ -0,0 +1,67 @@
+{% load i18n corpus_extras %}
+{% spaceless %}
```

Comment:
Should we not show any pagination when `is_paginated` is false? (i.e., only one page of results)

Maybe this is a question for @gissoo

Reply:
@gissoo What do you think here? I'm leaning towards not showing pagination controls when there's only one page.

Reply:
I've thought about this more — I don't think we should show pagination controls when there's only one page. Only thing to be aware of is we don't want the page content to shift too much if pagination appears/disappears depending on how the search is filtered, so will probably want to do something to keep the spacing the same even when pagination is not included.